helm/linkerd2-app/templates/hooks/hook-pre-install-identity-certs.yaml

```diff
@@ -29,11 +31,57 @@ metadata:
     "helm.sh/hook": pre-install
     "helm.sh/hook-weight": "-5"
 spec:
-  secretName: linkerd-identity-issuer
   commonName: identity.{{.Release.Namespace}}.{{.Values.global.clusterDomain}}
```

Comment:
Is the `svc.` part missing in FQDN here?

Reply:
good point, but seemingly not - there's no `svc` in the commonnames used here: https://linkerd.io/2/tasks/automatically-rotating-control-plane-tls-credentials/